### PR TITLE
[MO] - [removing module] -> test-containers migration

### PR DESCRIPTION
**Pull request by see-quick**
Signed-off-by: morsak <xorsak02@stud.fit.vutbr.cz>

### Description

This PR removing module test-containers from the strimzi-operator repository and moving to strimzi-test-container repository.

### CODE REVIEW

**Comment by see-quick** [comment]
from Travis `class file has wrong version 55.0, should be 52.0` -  the test-container from the repo another must build in that version to be able to merge it.

**Comment by ppatierno** [comment]
@see-quick you are removing but I don't see any reference to use it from the JAR built trough the new repo. Is it intentional? isn't it already used today so we need it?

**Comment by see-quick** [comment]
I am not sure if I understand this correctly.

File: pom.xml [hunk 1]
```diff
@@ -129,6 +128,14 @@
...
     </properties>
 
+    <repositories>
+        <repository>
+            <id>test-container</id>
+            <name>Strimzi Kafka Container repository</name>
+            <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
+        </repository>
+    </repositories>
+
```
Comment (scholzj) [inline pom.xml:138]: Why do we need to add this? This seems to introduce circular dependency -> if you use the test containers here, but building a new version of test-containers depends on the container images build by this repo. If we need to have this kind of dependency I'm not sure we can really move the test container to separate repo.

**Pull request closed without merging**